packages/dx-react-scheduler/src/plugins/view-switcher.tsx

```diff
@@ -35,7 +39,8 @@ class ViewSwitcherBase extends React.PureComponent<ViewSwitcherProps> {
               setCurrentViewName,
             }) => (
               <Switcher
-                currentView={currentView}
+                currentView={this.getMemoCurrentView(currentView.name,
```

Comment:
 Should we use this function here?

Reply:
We need to use this function because ViewSwitcher renders every time a date is changed. We need to use the function in order to avoid unnecessary renders